### ISSUE TITLE
[PRE REVIEW]: taurenmd: A command-line interface for analysis of Molecular Dynamics simulations.

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2142]
**Submitting author:** @joaomcteixeira (<a href="http://orcid.org/0000-0002-9113-0622">João M.C. Teixeira</a>)
**Repository:** <a href="https://github.com/joaomcteixeira/taurenmd" target ="_blank">https://github.com/joaomcteixeira/taurenmd</a>
**Version:** v0.8.9
**Editor:** @richardjgowers
**Reviewers:** @amritagos, @luthaf
**Managing EiC:** Kevin M. Moerman

**Author instructions**

Thanks for submitting your paper to JOSS @joaomcteixeira. **Currently, there isn't an JOSS editor assigned** to your paper.

@joaomcteixeira if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.84  T=0.14 s (590.2 files/s, 59733.3 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Python                          36           1272           1397           3162
SVG                              2              1              1           1059
reStructuredText                32            258            150            431
INI                              1             15              0            169
TeX                              1             15              0            140
YAML                             4             14             17            136
Markdown                         6             41              0            118
TOML                             1              0              0              5
-------------------------------------------------------------------------------
SUM:                            83           1616           1565           5220
-------------------------------------------------------------------------------


Statistical information for the repository '2142' was gathered on 2020/03/04.
The following historical commit information, by author, was found:

Author                     Commits    Insertions      Deletions    % of changes
Joao MC Teixeira               131          7350           3006           77.51
joaomcteixeira                  72          2245            760           22.49

Below are the number of rows from each author that have survived and are still
intact in the current revision:

Author                     Rows      Stability          Age       % in comments
Joao MC Teixeira           5831           79.3          2.3                8.03
```

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.1038/267585a0 is OK
- 10.1002/wcms.1449 is OK
- 10.1021/ct400341p is OK
- 10.1002/jcc.21787 is OK
-  10.25080/Majora-629e541a-00e  is OK
- 10.1016/j.bpj.2015.08.015 is OK
- 10.1002/jcc.23753 is OK
- 10.12688/f1000research.17456.1 is OK
- 10.1371/journal.pcbi.1005659 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.02142/joss.02142/10.21105.joss.02142.pdf)

**Comment by Kevin-Mattheus-Moerman** [comment]
@majensen @will-rowe @csoneson are you able to edit this submission?

**Comment by arfon** [comment]
@Kevin-Mattheus-Moerman - we have a new editor joining imminently who this would be perfect for. Can we hold off on this for a few days until I've had a chance to add them to the editorial team?

**Comment by Kevin-Mattheus-Moerman** [comment]
@arfon, yes sure that sounds good.

**Comment by arfon** [comment]
:wave: @richardjgowers - I think this also be an excellent paper for you to take on. What do you think?

/ cc @jedbrown

**Comment by richardjgowers** [comment]
@arfon yep

**Comment by Kevin-Mattheus-Moerman** [comment]
@whedon assign @richardjgowers as editor

**Comment by whedon** [comment]
OK, the editor is @richardjgowers

**Comment by richardjgowers** [comment]
@luthaf any chance you can help review this?

**Comment by richardjgowers** [comment]
@amritagos are you free to review this?

**Comment by Kevin-Mattheus-Moerman** [comment]
@richardjgowers a colleague of mine from NUIG @MahdiTavakol (a PhD student in our lab who works on molecular dynamics simulations) is keen to join this review. Add him if you can use an additional reviewer.

**Comment by amritagos** [comment]
@richardjgowers I would be happy to review this.

**Comment by Luthaf** [comment]
@richardjgowers I can do it, or leave it MahdiTavakol if they want to do this!

**Comment by richardjgowers** [comment]
@whedon add @amritagos as reviewer

**Comment by whedon** [comment]
OK, @amritagos is now a reviewer

**Comment by richardjgowers** [comment]
@whedon add @luthaf as reviewer

**Comment by whedon** [comment]
OK, @luthaf is now a reviewer

**Comment by richardjgowers** [comment]
@mahditavakol are you free to review this?

**Comment by arfon** [comment]
Dear authors and reviewers

We wanted to notify you that in light of the current COVID-19 pandemic, JOSS has decided to suspend submission of new manuscripts and to handle existing manuscripts (such as this one) on a "best efforts basis". We understand that you may need to attend to more pressing issues than completing a review or updating a repository in response to a review. If this is the case, a quick note indicating that you need to put a "pause" on your involvement with a review would be appreciated but is not required.

Thanks in advance for your understanding.

_Arfon Smith, Editor in Chief, on behalf of the JOSS editorial team._

**Comment by joaomcteixeira** [comment]
Dear editor and reviewers,

Is there any news regarding restarting operations at JOSS after the suspension announced by @arfon referring to COVID-19? These are difficult times and I most deeply hope all you are in safe conditions.

A simple question: in parallel to this submission, I have continued developing the software. Should I update any data on the manuscript if a new PR is merged?

Thanks in advance,

**Comment by arfon** [comment]
> A simple question: in parallel to this submission, I have continued developing the software. Should I update any data on the manuscript if a new PR is merged?

Given that the software has yet to go out for review I think it's fine to make small changes to the submission paper. Once the reviewers are working on the review I think it would be good to keep things stable.

**Comment by joaomcteixeira** [comment]
Thanks @arfon,
I look forward to the continuation of the review process.
Kind Regards,

**Comment by arfon** [comment]
:wave: @richardjgowers - given there are two reviewers identified already I am going to move this forward to review. If you decide we need a third reviewer then we can add them during the review process later.

**Comment by arfon** [comment]
@whedon start review

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/2175.

**Comment by arfon** [comment]
@amritagos, @Luthaf, @joaomcteixeira - see you all over in #2175 where the actual review will take place.
